xla/hlo/parser/hlo_parser_test.cc

```diff
@@ -2892,7 +2892,7 @@ TEST_F(HloParserTest, Empty) {
 
 TEST_F(HloParserTest, Garbage) {
   const std::string original =
-      "HloModule thi$ str1ng makes# N0 sen$e @all!*&^%$";
+      "HloModule this str1ng makes# N0 sen$e @all!*&^%$";
```

Comment:
This typo is intentional. Could you please change back?

Reply:
changed as suggested